pkg/volumeclaimbinder/persistent_volume_claim_binder.go

```diff
@@ -158,6 +158,14 @@ func syncVolume(volumeIndex *persistentVolumeOrderedIndex, binderClient binderCl
 
 	// available volumes await a claim
 	case api.VolumeAvailable:
+		// TODO:  remove api.VolumePending phase altogether
```

Comment:
My feeling is "not very breaking" which means we could drop that phase after 1.0 without much impact